cmd/argocd/commands/app.go

```diff
@@ -53,6 +54,8 @@ import (
 	"github.com/argoproj/argo-cd/v2/util/text/label"
 )
 
+type void struct{}
```

Comment:
No need to create a type for such a small definition. In this case I think it is better to just have `struct{}` directly in the rest of the code for clarity.

Reply:
Done the change